### ISSUE TITLE
Replace exec statement with exec() function in docs/core/benchmarks

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8419]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8419|
|Type|enhancement|
|Created|2016-06-04 05:30:55Z|
|Branch|https://github.com/twisted/twisted/tree/8419-rodrigc-exec|

See: https://docs.python.org/3.0/whatsnew/3.0.html#removed-syntax

**Removed keyword: exec() is no longer a keyword; it remains as a function. (Fortunately the function syntax was also accepted in 2.x.) **

In Python 2, you can do this:

```
    exec "foobar"    # WORKS, exec can be invoked as a statement
    exec("foobar")   # WORKS, exec can be invoked as a function
```

In Python 3, you can only do:

```
    exec "foobar"     # SyntaxError !
    exec("foobar")    # WORKS
```

The use of exec as a statement need to be changed to function form
to work with Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8419 8419
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8419_rodrigc_exec 8419-rodrigc-exec
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465018255445269 1465018255445269
changetime__1465661431102322 1465661431102322
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/139

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. thanks!

waiting for build results and will merge.


Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset b2373ffc0c02c7eaaa04bddd654adbf2097cc7ac
```
#!CommitTicketReference repository="" revision="b2373ffc0c02c7eaaa04bddd654adbf2097cc7ac"
Merge 8419-rodrigc-exec: Use exec function in docs/core/benchmarks.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8419
```
